bedrock/mozorg/templates/mozorg/about/policy/transparency/jul-dec-2023.html

```diff
@@ -0,0 +1,372 @@
+      <a href="https://blog.mozilla.org/netpolicy/files/2023/11/eIDAS-Industry-Letter.pdf">open letter</a>,
+      conducting meetings with members of the European Council, Parliament and Commission,
+      and engaging with civil society <a href="https://nce.mpi-sp.org/index.php/s/cG88cptFdaDNyRr">allies</a>.
+      Through these efforts, we were able to help persuade the European Parliament to introduce
+      a cybersecurity clarification to Article 45 as part of its adopted public
+      <a href="https://www.europarl.europa.eu/doceo/document/A-9-2023-0038-AM-007-007_EN.pdf">position</a>
+      on the legislation which will largely mitigate the security and privacy risks.
+      We also reiterated our <a href="https://blog.mozilla.org/netpolicy/2023/07/13/european-parliaments-version-of-the-cra-threatens-cybersecurity-and-open-source-development/">concerns</a>
+      on the Cyber Resilience Act (CRA) and engaged in conversations to protect the interests
+      of the open source community. We also were quoted in <a href="https://netzpolitik.org/2023/gefaehrlicher-praezedenzfall-neues-gesetz-in-frankreich-will-browser-zur-zensur-verpflichten/">media</a>
+      about browser support for global privacy control in Germany and ran a
+      <a href="https://foundation.mozilla.org/en/campaigns/sign-our-petition-to-stop-france-from-forcing-browsers-like-mozillas-firefox-to-censor-websites/">campaign</a>
+      on France’s efforts to make browsers the
+      <a href="https://thenextweb.com/news/french-law-threatens-free-internet-warns-mozilla">vectors</a>
+      of website blocking.
+    </p>
+
+    <p>
+      In the United States, we called for <a href="https://blog.mozilla.org/netpolicy/2023/07/27
+      /mozilla-calls-on-congress-to-reform-fisa/">reform</a> of Section 702 of the Foreign
+      Intelligence Surveillance Act (FISA) and organised a builders <a href="https://blog.
+      mozilla.org/netpolicy/2023/12/08/mozilla-and-allies-say-no-to-surveillance-blank-check-
+      in-ndaa-yes-to-strong-surveillance-protections/">letter</a> in
+      line with our long-standing <a href="https://wiki.mozilla.org/Netpolicy/
+      actions#Surveillance">advocacy</a> on the issue. We also submitted <a href="https://blog.
+      mozilla.org/netpolicy/2023/08/11/mozilla-supports-updates-to-the-health-breach-
+      notification-rule/">comments</a> on the FTC’s Notice of Proposed Rulemaking for
+      the Health Breach Notification
+      Rule (HBNR) and <a href="https://blog.mozilla.org/netpolicy/2023/08/18/
+      mozilla-applauds-cfpb-for-taking-on-the-data-broker-ecosystem/">applauded</a> the
+      Consumer Financial Protection Bureau (CFPB) for taking on the data broker ecosystem.
+      Finally, we also weighed in with our <a href="https://blog.mozilla.org/netpolicy/
+      2024/01/11/mozilla-weighs-in-on-state-comprehensive-privacy-proposals/">thoughts</a>
+      on the state privacy proposals in Massachusetts and Maine.
```

Comment:
**Issue:** these links aren't working as there's a space after the line break in `https://blog.`